app/assets/stylesheets/views/_landing_page/main-navigation.scss

```diff
@@ -0,0 +1,30 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-main-navigation {
+  padding-top: govuk-spacing(3);
+  padding-bottom: govuk-spacing(6);
+  padding-left: 0;
+  padding-right: 0;
+  margin: 0;
+
+  .app-b-main-navigation__listitem:last-of-type {
+    margin-right: 0;
+  }
+}
+
+.app-b-main-navigation__listitem {
+  list-style: none;
+  display: inline-block;
+  margin-right: govuk-spacing(9);
+  margin-top: govuk-spacing(3);
```

Comment:
Nitpick: the spacing around these elements is inconsistent - `.app-b-main-navigation` has different padding top and bottom, but then the spacing is evened out by having top margin on the list items. Is there a reason it's like this, or could we just do

```CSS
.app-b-main-navigation {
  padding: govuk-spacing(6) 0;
}

.app-b-main-navigation__listitem {
  // no margin top
}
```

Reply:
Good spot - this was actually intentional for future proofing. If the block ends up with enough links to cause overflow, the rows of links will need space between them, and we need to achieve that while still having the balance of overall top and bottom spacing, which is why I wrote it in this way.

<img width="766" alt="image" src="https://github.com/user-attachments/assets/2a20cf2e-672f-4998-af53-e36bdfae3b9b">

Reply:
THAT - is nicely done sir. Excellent. I withdraw my comment 😁